uisdk/src/test/java/com/karhoo/uisdk/util/CurrencyExtensionsTest.kt

```diff
@@ -85,13 +85,13 @@ class CurrencyExtensionsTest {
     @Test
     fun `when formatting Japanese Yen prices with no symbol, should display 0 fraction digits`() {
         val formattedString = jpyCurrency.intToPriceNoSymbol(price = 180, locale = Locale.UK)
-        assertEquals("180", formattedString)
+        assertEquals("180.", formattedString)
```

Comment:
Why is there a dot at the end of the price without having decimals?

Reply:
It seems the current intToPriceNoSymbol returns amounts like "180, " for french and as a hotfix I reverted to the old implementation, It might need to be revisited again. I have tried to fix the unit tests and I had to use "180."